Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu
+
+# set up timezone
+RUN echo "America/New_York" > /etc/timezone
+
+# install required packages
+RUN apt-get update \
+&& DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    python3 \
+    python3-dev \
+    python3-pip \
+    ffmpeg python3-tk \
+&& rm -rf /var/lib/apt/lists/*
+
+# setup locales and language to utf-8
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+
+# create a new directory
+RUN mkdir /MobOff
+
+# copy current directory to /MobOff 
+COPY . /MobOff
+
+# run the installation script
+RUN python3 setup.py install
+
+# run bash
+CMD ["/bin/bash"]
```

Comment:
This is a good start. But as we are building docker specifically for *MobOff*, we should be able run *MobOff download* at the start of the container. For this, we need to use `ENTRYPOINT` in the Dockerfile. Also, in our case we need to initialize first. So we should use Environmental Variable that can be passed through _docker run_

Reply:
I agree. But even before `moboff download` we have to `initialise` it. I think we should dump the `moboff initialise` command and pass the API key as environment variable.

Reply:
Yes. Exactly! We need to use Environmental variables like I said in my comment.